services/merak-topo/database/entities.go

```diff
@@ -26,37 +26,44 @@ const (
 )
 
 type Nic struct {
+	Id   string `json:"id"`
 	Intf string `json:"intf"`
 	Ip   string `json:"ip"`
 	Mac  string `json:"mac"`
 }
 
 type Vnode struct {
-	Id          string                   `json:"id"`
-	Name        string                   `json:"name"`
-	Nics        []Nic                    `json:"nics"`
-	Flinks      []map[string]interface{} `json:"flinks"`
-	ContainerIp string                   `json:"containerip"`
-	Status      ServiceStatus            `json:"status"`
+	Id          string        `json:"id"`
+	Type        string        `json:"type"`
+	Name        string        `json:"name"`
+	Nics        []Nic         `json:"nics"`
+	Flinks      []Vlink       `json:"flinks"`
+	ContainerIp string        `json:"containerip"`
+	Status      ServiceStatus `json:"status"`
 }
 
-type Vport struct {
-	Id   string `json:"id"`
-	Name string `json:"name"`
-	Intf string `json:"nic"`
-	Ip   string `json:"ip"`
-}
+// type Vport struct {
+// 	Id   string `json:"id"`
+// 	Name string `json:"name"`
+// 	Intf string `json:"nic"`
+// 	Ip   string `json:"ip"`
+// }
```

Comment:
@zzengcs If you don't need it, please remove it from the code.

Reply:
addressed